src/config.rs

```diff
@@ -48,7 +48,6 @@ impl fmt::Display for AddressError {
 }
 
 /// Newtype for an address that is parsed as `String`
-///
```

Comment:
Removing this will stick the lines together like "... as `String` The main point ...", that seems unintentional.